File: C++/633. Sum of Square Numbers.cpp
class Solution {
public:
    bool judgeSquareSum(int c) {
        double i,a;
        for(i=0;i*i<=c;i++){
            a=sqrt(c-i*i);
            if(a==(int)a)
            return true;
        }
        return false;
    }
};

File: JAVA/633. Sum of Square Numbers.java
class Solution {
    public boolean judgeSquareSum(int c) {
        long l=0,r=(long )Math.sqrt(c),s=0;
        while(l<=r){
            s=l*l+r*r;
            if(s==c)
            return true;
            else if(s>c)
            r--;
            else
            l++;
        }
        return false;
    }
}